core/src/main/java/org/apache/gravitino/metalake/MetalakeManager.java

```diff
@@ -116,14 +165,13 @@ public BaseMetalake createMetalake(
       NameIdentifier ident, String comment, Map<String, String> properties)
       throws MetalakeAlreadyExistsException {
     long uid = idGenerator.nextId();
-    StringIdentifier stringId = StringIdentifier.fromId(uid);
```

Comment:
Why do we remove this?

Reply:
ince the uid was already in the metalake entity field and I don't see that uid being used in properties, I removed it.